### ISSUE TITLE
Error: Error watching file for changes: EMFILE

### ISSUE THREAD

**Issue by kirkstrobeck** [facebook/watchman#458]
See https://github.com/facebook/jest/issues/3436

---

I know others have posted this issue, but it seems that they are aging and using old versions. I have tried everything from other issues. 

Here is the error I get:

```bash
$ jest --watch # or with $ jest --watch --no-watchman
2017-05-02 09:49 node[8980] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-05-02 09:49 node[8980] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-05-02 09:49 node[8980] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: Error watching file for changes: EMFILE
    at exports._errnoException (util.js:1050:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1376:11)
error Command failed with exit code 1.
```

Versions:

- node: v7.9.0
- jest: 19.0.2
- watchman: 4.7.0 (installed via brew)
- mac: Sierra 10.12.4 (16E195)

**Comment by kirkstrobeck** [comment]
May not be watchman

**Comment by yogieputra8** [comment]
Hi, I'm facing the same issue. What is the answer?

**Comment by jabezGit** [comment]
I was running into the same issue, and this solved it: [#910](https://github.com/facebook/react-native/issues/910)
